list_dependencies/test/test.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
```

Comment:
you should also add this to the root test.sh ! 
https://github.com/tilt-dev/tilt-extensions/blob/master/test.sh

Reply:
Done, thank you!